### PR TITLE
Nxlog exchange conf

### PR DESCRIPTION
**Pull request by melvin-suter**

### CODE REVIEW

File: files/nxlog_exchange_2010.conf [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+<Route 2>
+   Path        in_sec => out
+</Route>
+ 
+<Route 3>
+   Path        in_sys => out
+</Route>
+  
+ 
+<Route 2>
```
Comment (joe-speedboat) [inline files/nxlog_exchange_2010.conf]: Is there a reason to reuse route names?
May be it should be <Route 4>
```suggestion
<Route 4>
```

File: files/nxlog_exchange_2010.conf [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+   Path        messagetracking => out
+</Route>
+ 
+<Route 3>
```
Comment (joe-speedboat) [inline files/nxlog_exchange_2010.conf]:
```suggestion
<Route 5>
```

File: files/nxlog_exchange_2010.conf [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+   Path        connectivity => out
+</Route>
+
+<Route 4>
```
Comment (joe-speedboat) [inline files/nxlog_exchange_2010.conf]:
```suggestion
<Route 6>
```

File: files/nxlog_exchange_2010.conf [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+   Path        smtp_receive => out
+</Route>
+
+<Route 5>
```
Comment (joe-speedboat) [inline files/nxlog_exchange_2010.conf]:
```suggestion
<Route 6>
```

File: files/nxlog_exchange_2016.conf [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+<Route 2>
+   Path        in_sec => out
+</Route>
+ 
+<Route 3>
+   Path        in_sys => out
+</Route>
+  
+ 
+<Route 2>
```
Comment (joe-speedboat) [inline files/nxlog_exchange_2016.conf]:
```suggestion
<Route 4>
```

File: files/nxlog_exchange_2016.conf [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+   Path        messagetracking => out
+</Route>
+ 
+<Route 3>
```
Comment (joe-speedboat) [inline files/nxlog_exchange_2016.conf]:
```suggestion
<Route 5>
```

File: files/nxlog_exchange_2016.conf [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+   Path        connectivity => out
+</Route>
+
+<Route 4>
```
Comment (joe-speedboat) [inline files/nxlog_exchange_2016.conf]:
```suggestion
<Route 6>
```

File: files/nxlog_exchange_2016.conf [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+   Path        smtp_receive => out
+</Route>
+
+<Route 5>
```
Comment (joe-speedboat) [inline files/nxlog_exchange_2016.conf]:
```suggestion
<Route 7>
```

**Comment by melvin-suter** [comment]
My bad, changed it localy to test but didn't update it in the repo.

File: files/nxlog_exchange_2010.conf [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+ 
+<Route 5>
+   Path        connectivity => out
+</Route>
+
+<Route 6>
+   Path        smtp_receive => out
+</Route>
+
+<Route 6>
```
Comment (joe-speedboat) [inline files/nxlog_exchange_2010.conf]:
```suggestion
<Route 7>
```
Da hab ich glaub einen vergessen :-(

**Comment by melvin-suter** [comment]
This version should stay constant for a while. Merging into master

**Pull request merged**